pkg/exchange/binance/exchange.go

```diff
@@ -33,13 +32,9 @@ const BNB = "BNB"
 const DefaultDepthLimit = 5000
 
 const BinanceUSBaseURL = "https://api.binance.us"
-const BinanceTestBaseURL = "https://testnet.binance.vision"
 const BinanceUSWebSocketURL = "wss://stream.binance.us:9443"
 const WebSocketURL = "wss://stream.binance.com:9443"
-const WebSocketTestURL = "wss://testnet.binance.vision"
-const FutureTestBaseURL = "https://testnet.binancefuture.com"
```

Comment:
these are still required because we want to replace adshao/binance with the requestgen api

Reply:
ok, I'll keep it

Reply:
all constants containing test url are kept, done